starter-code/src/components/AddFood.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
+
+// {
+//     "name": "Pizza",
+//     "calories": 400,
+//     "image": "https://i.imgur.com/eTmWoAN.png",
+//     "quantity": 0
+//   },
+class AddFood extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: "",
+      calories: "",
+      image: "",
+      quantity: ""
+    };
+  }
+
+  //   handleFormSubmit = event => {
+  //     event.preventDefault();
+  //     this.props.addTheMovie(this.state);
+  //     this.setState({
+  //       name: "",
+  //       calories: "",
+  //       image: "",
+  //       quantity: ""
+  //     });
+  //   };
+  handleFormSubmit = event => {
+    event.preventDefault();
+    this.props.addNewFood(this.state);
+  };
+  onFormFieldChange = event => {
+    const inputName = event.target.name;
+    this.setState({
+      [inputName]: event.target.value
+    });
+  };
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.handleFormSubmit}>
+          <label>name:</label>
```

Comment:
Using the [for attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes) of `<label>` will make your app more accessible.